docs/en/query_language/functions/string_functions.md

```diff
@@ -217,17 +217,119 @@ Result:
 └───────────────────────────────────┘
 ```
 
-## trimLeft(s)
+## trimLeft {#trimleft}
 
-Returns a string that removes the whitespace characters on left side.
+Removes whitespace from the beginning of a string. Whitespace in this context is all the whitespace characters (space, tab, no-break space, etc.).
 
-## trimRight(s)
+**Syntax** 
 
-Returns a string that removes the whitespace characters on right side.
+```sql
+trimLeft()
+```
+
+Alias: `ltrim`.
+
+**Parameters** 
+
+- `string` — string to trim. [String](https://clickhouse.yandex/docs/en/data_types/string/).
+
+**Returned value**
+
+A string without any heading whitespaces.
+
+Type: `String`.
+
+**Example**
+
+Query:
+
+```sql
+SELECT concat(trimLeft('     Hello,     '), ' world!')
```

Comment:
why do we need `concat` in this example? It'd be simpler with just string literal